backend/.env.test_e2e

```diff
@@ -0,0 +1,9 @@
+# Server
```

Comment:
this is just a suggestion and can be closed:
What about putting these things in the `docker-compose.test.yml` file?

Reply:
This is kept in mind for the upcumming test workflow refactoring.